index.html

```diff
@@ -85,7 +84,35 @@ <h1>I'm Jonathan Glad to see you</h1>
         </div>
     </section>
 </main>
-</div>
-</div>
+<section class="project-section">
+  <div class="project-container">
+      <div class="project-card">
+          <img src="./assets/Snapshoot Portfolio-2.png" alt="project card image" />
+          <div class="prject-card-info flex column">
+              <h2 class="project-title">Tonic</h2>
+              <div class="project-meta flex">
+                  <p class="bold">Canopy</p>
+                  <div class="round"></div>
+                  <p>Backend</p>
+                  <div class="round"></div>
+                  <p>2015</p>
+              </div>
+              <div class="project-note">
+                  <p>
+                      A daily selection of privately personalized reads; no accounts
+                      or sign-ups required.
+                  </p>
+              </div>
+              <div class="project-tag">
+                  <ul class="project-skills flex">
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                  </ul>
+              </div>
+              <button class="see-project-btn">See Project</button>
```

Comment:
- [ ] Kindly make your hover and pressed state match the Figma template. You are expected to stick to the Figma design as much as possible.
### Figma 
![image](https://user-images.githubusercontent.com/94551484/219360749-570c35fe-14d9-4685-94ff-9c4b11c32b38.png)